packages/common/src/reducers/messages.js

```diff
@@ -179,6 +179,7 @@ const systemMessageTypes = ["member-joined"];
 
 const deriveMessageType = (message) => {
   switch (message.type) {
+    case undefined:
```

Comment:
this will help both with the messages created by the client, but also in case older messages (without the `type` on DB) were to come without a `type` field.